css-brochure-form/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h2>LEARN MORE</h2>
+  <form>
+    <label for="email">COMPANY EMAIL</label>
+    <div>
+      <input type="text" name="email" id="email" value="sally@company.co">
+    </div>
+    <label for="name">NAME</label>
+    <div>
+      <input type="text" name="name" id="name" value="Sally Smith">
+    </div>
+    <label for="where">WHERE ARE YOU HIRING?</label>
+    <div>
+      <input type="text" name="where" id="where" value="City, Country">
```

Comment:
Instead of pre-filling the form with values, the text you see in the inputs are actually `placeholder` attributes.